### PR TITLE
fix(referrers): fix some conformance issues

### PR DESCRIPTION
**Pull request by rchincha**
Signed-off-by: Ramkumar Chinchani <rchincha@cisco.com>

**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by peusebiu — APPROVED** [review]
lgtm

File: pkg/api/routes.go [hunk 1]
```diff
@@ -482,26 +482,14 @@ func (rh *RouteHandler) GetReferrers(response http.ResponseWriter, request *http
...
+	// filter by artifact type (more than one can be specified)
+	artifactTypes := request.URL.Query()["artifactType"]
 
-	rh.c.Log.Info().Str("digest", digest.String()).Str("artifactType", artifactType).Msg("getting manifest")
+	rh.c.Log.Info().Str("digest", digest.String()).Interface("artifactType", artifactTypes).Msg("getting manifest")
 
 	imgStore := rh.getImageStore(name)
 
-	referrers, err := getReferrers(request.Context(), rh, imgStore, name, digest, artifactType)
+	referrers, err := getReferrers(request.Context(), rh, imgStore, name, digest, artifactTypes)
```
Comment (andaaron) [inline pkg/api/routes.go:492]: Where is the artifactTypes variable cast to a list of strings? Or that is not necessary?

**Review by andaaron — APPROVED** [review]
Looks good, but do we have tests with multiple references sent to the rest API?

File: pkg/api/routes.go (hunk 1 above)
Comment (rchincha) [reply pkg/api/routes.go:492]: Not, needed but agreed, definitely needs a test.

**Review by andaaron — APPROVED** [review]

**Review by andaaron — APPROVED** [review]

**Pull request merged**